File: Java Script/Conditional Statements - Lab/areaOfFigures.js
function area(input) {
    let figure = (input[0]);
    if (figure === "square") {
        let side = Number(input[1]);
        console.log((side * side).toFixed(3));
    } else if (figure === "rectangle"){
        let sideOne = Number(input[1]);
        let sideTow = Number(input[2]);
        let areaRectangle = sideOne * sideTow;
        console.log((areaRectangle).toFixed(3));
    } else if (figure === "circle"){
        let r = Number(input[1]);
        let areaCircle = Math.PI * Math.pow(r,2)
        console.log((areaCircle).toFixed(3))
    } else if (figure === "triangle"){
        let base = Number(input[1]);
        let height = Number(input[2]);
        let areaTriangle = (base * height) / 2;
        console.log((areaTriangle).toFixed(3));
    }
}
area(["square", "5"]);

File: Python/python_PB/Conditional Statements - Lab/area_of_figures.py
from math import pi
figure = input()
if figure == "square":
    side = float(input())
    result = side * side
elif figure == "rectangle":
    side1 = float(input())
    side2 = float(input())
    result = side2 * side1
elif figure == "circle":
    diameter = float(input())
    result = (diameter **2) * pi
elif figure == "triangle":
    side = float(input())
    height = float(input())
    result = (side * height) / 2
print(f"{result:.3f}")
